0-ad-history-mod-package/simulation/templates/template_structure_military_elephant_stable.xml

```diff
@@ -31,6 +31,10 @@
     <Requirements>
       <Techs datatype="tokens">phase_city</Techs>
     </Requirements>
+    <History>
+        Humans have for a long time kept elephants; the earliest evidence dates back to the year 2000 BC. They were, however, unlike other animals, not domesticated. Because breeding in general proved nearly impossible due to their enormous size, complex social behavior, and long gestation, most elephants were instead captured from the wild.
```

Comment:
```suggestion
       Humans have for a long time kept elephants; the earliest evidence dates back to the year 2000 BC. However, unlike other animals, they were not domesticated because breeding  proved nearly impossible due to their enormous size, complex social behavior, and long gestation. Instead, most elephants were instead captured from the wild.
```